### PR TITLE
Added tokenomics code to new workflow

### PR DESCRIPTION
**Pull request by Jayashsatolia403**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -172,8 +172,8 @@ jobs:
           deploy_report_page: true
           archive_results: true
           run_flaky_tests: true
-          run_api_system_tests: true
-          run_cli_system_tests: true
+          run_api_system_tests: false
+          run_cli_system_tests: false
```
Comment (Kishan-Dhakan) [inline .github/workflows/ci.yml:176]: This ci.yml is the workflow file for manual runs and automated runs on every commit, we don't want to set them to false here. What you want to do is, create a separate workflow file, that runs manually, and only runs tokenomics suite. You can read on github workflows or check out or workflow files to see how they work.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Partial review: we don't want to turn off other tests in the main runs, we want to keep tokenomics suite disabled in the main run, and run only tokenomics suite when we need (via a manual trigger or cron job)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Jayashsatolia403) [reply .github/workflows/ci.yml:176]: Yeah, I just turned them off to get check if my changes are working faster.

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | tokenomics_suite  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | tokenomics_suite_cicd_test  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request closed without merging**